.github/pull_request_template.md

```diff
@@ -1,52 +1,18 @@
 <!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
 v                               ✰  Thanks for creating a PR! ✰    
-v    Before smashing the submit button please review the checkboxes.
-v    If a checkbox is n/a - please still include it but + a little note why
-v    If your PR doesn't close an issue, that's OK!  Just remove the Closes: #XXX line!
 ☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
-
+## Which issue does this close?
 Closes: #XXX
+## Purpose of the change
+*Add a description of the overall background and high level changes that this PR introduces*
```

Comment:
```suggestion
## Which issue does this close?

closes: #XXX

## Purpose of the change

> Add a description of the overall background and high level changes that this PR introduces*
```

In general, I think these templates should use comments, i.e. `>` as much as possible. What ends up happening is that authors forget or just ignore replacing the section and you end up with a PR that just has useless template text.